api/task_check_run.go

```diff
@@ -71,8 +71,8 @@ const (
 	TaskCheckDatabaseStatementSyntax TaskCheckType = "bb.task-check.database.statement.syntax"
 	// TaskCheckDatabaseStatementCompatibility is the task check type for statement compatibility.
 	TaskCheckDatabaseStatementCompatibility TaskCheckType = "bb.task-check.database.statement.compatibility"
-	// TaskCheckDatabaseStatementRequireWhere is the task check type for the WHRER clause requirement for UPDATE/DELETE.
-	TaskCheckDatabaseStatementRequireWhere TaskCheckType = "bb.task-check.database.statement.where.require"
+	// TaskCheckDatabaseStatementSchemaReview is the task check type for schema system review policy.
+	TaskCheckDatabaseStatementSchemaReview TaskCheckType = "bb.task-check.database.statement.schema-review"
```

Comment:
TaskCheckDatabaseStatementAdvise?

schema review is too specific. e.g. Data change is not schema and it requires enforcing the policy.

Advise is a bit general, while it's at least more accurate than schema review